src/test/resources/journeys.conf

```diff
@@ -15,52 +15,26 @@
 # Configure here your journeys. A journey is a sequence of requests at a certain load.
 
 journeys {
-
-  individual-journey = {
-    description = "Individual journey"
-    load = 9.1
-    parts = [
-        DigitalDisclosureIndividualJourney
-    ]
-  }
-
-  estate-journey = {
-    description = "Estate Journey"
-    load = 9.1
+  disclosure-company = {
+    description = "Submit a new disclosure (company)"
+    load = 1
```

Comment:
The load for the journey(s) need(s) to be configured using DDS load expectations from discovery phase of the project.